scripts/modify_chart_for_aws_marketplace.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+FILE="./stable/rad-plugins/values.yaml"
+
+GCR_REGISTRY_NAME="us.gcr.io/ksoc-public"
```

Comment:
We should replace this with our new ECR registry name

Reply:
fixed